src/platform/silabs/wifi/rs911x/platform/efx32_ncp_host.c

```diff
@@ -42,35 +42,34 @@
 #include "sl_board_control.h"
 #endif // SL_BOARD_NAME
 
-static bool dma_callback(unsigned int channel, unsigned int sequenceNo, void * userParam);
+#include "sl_spidrv_exp_config.h"
+#include "sl_spidrv_instances.h"
+#include "spidrv.h"
+
+#define LDMA_MAX_TRANSFER_LENGTH 4096
+#define LDMA_DESCRIPTOR_ARRAY_LENGTH (LDMA_MAX_TRANSFER_LENGTH / 2048)
+
+// use SPI handle for EXP header (configured in project settings)
+extern SPIDRV_Handle_t sl_spidrv_exp_handle;
+#define SPI_HANDLE sl_spidrv_exp_handle
+static uint8_t dummy_buffer[1800] = { 0 };

```

Comment:
Where is the 1800 coming from? Any calculation to explain that should be shown.

Reply:
1800 is the maximum size of command or data packet that can be transferred to the firmware from host.

Reply:
Can you add a define for the constant here? This would allow to provide information as to why 1800 is important.

Reply:
added